rename aad to EID and remove OSM (#111)
Azure/AKS-Landing-Zone-Accelerator: Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/03-eid.md

1. **This step only applies if you are creating a new EID group for this deployment. If you have one already existing and you are a part of it, you can skip this prerequisite, and follow the import portion of the instructions below**. 
   

>    An Microsoft Entra ID tenant to associate your Kubernetes RBAC Cluster API authentication to.

Reviewer: ```suggestion
   A Microsoft Entra tenant to associate your Kubernetes RBAC Cluster API authentication to.
```

   > :warning: The user or service principal initiating the deployment process _must_ have the following minimal set of Microsoft Entra ID permissions assigned:
   >

>    > * Microsoft Entra ID [User Administrator](https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference#user-administrator-permissions) is _required_ to create a "break glass" AKS admin Microsoft Entra ID Security Group and User. Alternatively, you could get your Microsoft Entra ID admin to create this for you when instructed to do so.

Reviewer: ```suggestion
   > * Microsoft Entra [User Administrator](https://learn.microsoft.com/entra/identity/role-based-access-control/permissions-reference#user-administrator-permissions) is _required_ to create a "break glass" AKS admin Microsoft Entra security group and user. Alternatively, you could get your Microsoft Entra ID admin to create this for you when instructed to do so.
```

>    >   * If you are not part of the User Administrator group in the tenant associated to your Azure subscription, please consider [creating a new tenant](https://learn.microsoft.com/en-us/entra/fundamentals/create-new-tenant#create-a-new-tenant-for-your-organization) to use while evaluating this implementation. The Microsoft Entra ID tenant backing your cluster's API RBAC does NOT need to be the same tenant associated with your Azure subscription.

Reviewer: ```suggestion
   >   * If you are not part of the User Administrator group in the tenant associated to your Azure subscription, please consider [creating a new tenant](https://learn.microsoft.com/entra/fundamentals/create-new-tenant#create-a-new-tenant-for-your-organization) to use while evaluating this implementation. The Microsoft Entra tenant backing your cluster's API RBAC does NOT need to be the same tenant associated with your Azure subscription.
```

> ## Create or Import Azure Microsoft Entra ID for AKS

Reviewer: ```suggestion
## Create or import Microsoft Entra groups for AKS
```
